publications/recommender.tex

```diff
@@ -29,17 +29,20 @@
 
 \title{Microsoft/Recommenders: Best practices for building industry-grade recommender system}
```

Comment:
question, do we want to name this as Microsoft/Recommenders or Microsoft Recommenders?

Reply:
This is the question I also have. We need to fix the name to call the repository throughout the paper.

Reply:
I also like Microsoft Recommenders